Apps/Playground/Win32/App.cpp

```diff
@@ -10,6 +10,11 @@
 #include <Shared/InputManager.h>
 
 #include <Babylon/AppRuntime.h>
+#if __has_include(<Babylon/AppRuntimeV8.h>)
+#include <Babylon/AppRuntimeV8.h>
+#define V8_ENGINE
+#endif
```

Comment:
I don't like these pragma conditions in this file. This specific one using Microsoft specific pragmas feels wrong to me. Also, what happens if someone uses V8 with Android? Does the inspector code work with Android?

Reply:
That's a good question-- I'll have to set up Android Studio to find out, but shouldn't be hard at all to test.

I'm not really sure how we can keep App.cpp engine-agnostic without using this conditional include or some sort of options object in the AppRuntime layer. Is there a different pattern I can use here to call V8-specific functions? Perhaps we could do something like [ReactNativeHost](https://github.com/microsoft/react-native-windows/blob/26eac15f92dca279f78e9280ebe0aa4db9d81749/vnext/Microsoft.ReactNative/ReactApplication.cpp#L135-L146)'s instance settings.

Reply:
yeah, i'm not sure there is a good solution. @syntheticmagus @ryantrem any thoughts?